PHPCompatibility/Util/Tests/CoreMethodTestFrame.php

```diff
@@ -68,7 +68,7 @@ protected function setUpPHPCS()
         $filename    = realpath(__DIR__) . \DIRECTORY_SEPARATOR . $subDir . \DIRECTORY_SEPARATOR . $className . '.inc';
         $contents    = file_get_contents($filename);
 
-        if (version_compare(PHPCSHelper::getVersion(), '2.99.99', '>')) {
+        if (version_compare(Helper::getVersion(), '2.99.99', '>')) {
```

Comment:
Not really a problem here, but it might be better to do version_compare with 3.0 and operator >= instead. Again, in this case not a problem, but it's best practice just in case a package every gets > 99 versions.

Reply:
I understand where you are coming from, but a compare like that with `3.0.0-alpha` or `3.0.0-RC` would fail as those are regarded as below 3.0.0: https://3v4l.org/CeIMA

Not relevant anymore now we're well past 3.0.0, but it was relevant when the adjustment was being made and `3.0.0` hadn't been released yet.

Reply:
Interesting, didn't know